src/devices/bus/mc10/multiports_ext.cpp

```diff
@@ -1,31 +1,34 @@
+// license:BSD-3-Clause
+// copyright-holders:S. Glaize
+
 /***************************************************************************
 
-    multiports_ext.cpp
+	multiports_ext.cpp
 
-    Emulation of the Alice Multiports Extension
+	Emulation of the Alice Multiports Extension
 
-    Features:
-        The extension provides an extension doubler and two joystick ports.
+	Features:
+		The extension provides an extension doubler and two joystick ports.
 
-        The extension also provides (for the whole Alice family and MC-10):
-        - 16K of RAM expansion ($5000-$8FFF)
-        - 64K of ROM expansion in two possible configurations:
-            - 8K of ROM between $1000 and $2FFF, as 8 banks (Cartridge mode).
-            - 16K of ROM between $C000 and $FFFF, as 4 banks (ROM mode).
+		The extension also provides (for the whole Alice family and MC-10):
+		- 16K of RAM expansion ($5000-$8FFF)
+		- 64K of ROM expansion in two possible configurations:
+			- 8K of ROM between $1000 and $2FFF, as 8 banks (Cartridge mode).
+			- 16K of ROM between $C000 and $FFFF, as 4 banks (ROM mode).
 
-        Only the RAM/ROM expansion is emulated here.
+		Only the RAM/ROM expansion is emulated here.
 
-    Banks are selected by writing to:
-        - $1000 to $1FFF in Cartridge mode (number of bank between 0 and 7)
-        - $C000 to $CFFF in ROM mode (number of bank between 0 and 3)
+	Banks are selected by writing to:
+		- $1000 to $1FFF in Cartridge mode (number of bank between 0 and 7)
+		- $C000 to $CFFF in ROM mode (number of bank between 0 and 3)
```

Comment:
This is going to end up getting changed back again by the `srcclean` tool – you should back it out to reduce churn (indentation inside multi-line comments only uses tabs up to the indentation level of the line containing the comment start delimiter).